File: ExerciciosC/maiores_que_oito.cpp
#include<conio.h>
#include<stdio.h>

main(){
	
	int numeros[19], maiores_oito;
	
	for(int i = 0; i <= 19; i++){
		
		printf("Digite o numero desejado = ");
		scanf("%d", &numeros[i]);
		
		if(numeros[i] > 8){
			maiores_oito++;
		}
		
	}
	
	printf("%d numeros digitados, sao maiores que oito.", maiores_oito);
	
}

File: ExerciciosJava/MaioresQueOito.java
import java.util.Scanner;

public class MaioresQueOito {

	public static void main(String[] args) {
		
		int numeros[] = new int[20], maiorQueOito = 0;
		
		Scanner s = new Scanner(System.in);

		for(int i = 0; i <= 19; i++) {
			
			System.out.println("Digite o numero desejado = ");
			numeros[i] = s.nextInt();
			
			if(numeros[i] > 8) {
				maiorQueOito++;
			}
			
		}
		
		System.out.println(maiorQueOito + " numeros digitados são maiores que oito");
		
	}

}
